dashboard/src2/data/logins.js

```diff
@@ -0,0 +1,24 @@
+import { createResource } from 'frappe-ui';
+
+let oauthLogins = createResource({
+	url: 'press.api.oauth.get_custom_oauths',
```

Comment:
avoid custom endpoints? can this be done using createListResource?

Reply:
@netchampfaris I tried with createListResource initially. But client.get_list is not allowed for guest request. Also the result for this would be same for every user so it's better we fetch from some shared server side cache instead of making db requests? I'm not sure if we can do this with resources api.

Reply:
@rutwikhdev In that case, this should be done in LoginSignup.vue. There is already an API call `press.api.account.signup_settings` that fetches data required for login page. Maybe add this there. I don't think it's going to be used anywhere so doesn't make sense to keep it in separate file.

Reply:
Alright. Will get rid of logins.js in that case.